cpp/cmake_modules/BuildUtils.cmake

```diff
@@ -331,6 +332,14 @@ function(ADD_ARROW_TEST REL_TEST_NAME)
   endif()
   get_filename_component(TEST_NAME ${REL_TEST_NAME} NAME_WE)
 
+  if(ARG_PREFIX)
+    set(TEST_NAME "${ARG_PREFIX}-${TEST_NAME}")
+  endif()
+
+  if (NOT ARG_TEST_GROUP)
```

Comment:
Add a comment what this is about? (what is a test group?)

Reply:
OK